From stern@rowland.harvard.edu Tue Sep 18 07:09:42 2007
From: Alan Stern <stern@rowland.harvard.edu>
Date: Tue, 18 Sep 2007 10:09:32 -0400 (EDT)
Subject: USB: unusual_devs update for Nokia 6131
To: Greg KH <greg@kroah.com>, Phil Dibowitz <phil@ipom.com>
Cc: Juan Ignacio Cherrutti <juan.i.cherrutti@accenture.com>,  USB Storage list <usb-storage@lists.one-eyed-alien.net>
Message-ID: <Pine.LNX.4.44L0.0709181008040.3629-100000@iolanthe.rowland.org>


This patch (as991) updates the unusual_devs entry for the Nokia 6131
phone.  As reported by Juan Ignacio Cherrutti, there's new firmware
available but it still has the same old transfer-size limit.

Signed-off-by: Alan Stern <stern@rowland.harvard.edu>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/storage/unusual_devs.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/usb/storage/unusual_devs.h
+++ b/drivers/usb/storage/unusual_devs.h
@@ -198,7 +198,7 @@ UNUSUAL_DEV(  0x0421, 0x044e, 0x0100, 0x
 		US_FL_IGNORE_RESIDUE | US_FL_FIX_CAPACITY ),
 
 /* Reported by Bardur Arantsson <bardur@scientician.net> */
-UNUSUAL_DEV(  0x0421, 0x047c, 0x0370, 0x0370,
+UNUSUAL_DEV(  0x0421, 0x047c, 0x0370, 0x0610,
 		"Nokia",
 		"6131",
 		US_SC_DEVICE, US_PR_DEVICE, NULL,
